src/commands/music/play.ts

```diff
@@ -0,0 +1,26 @@
+import {
+  ChatInputCommandInteraction,
+  GuildMember,
+  SlashCommandBuilder,
+} from 'discord.js';
+import { play } from 'services/music.player.service';
+import logger from 'utils/logger';
+
+export default {
+  data: new SlashCommandBuilder()
+    .setName('play')
+    .addStringOption((option) =>
+      option
+        .setName('song')
+        .setDescription('Name of the song or a URL')
```

Comment:
what URLs does it accept? only youtube? probably should mention that

Reply:
Tru, it's Spotify, Youtube and Deezer

Reply:
Deezer deez nuts